ignite/contrib/handlers/base_logger.py

```diff
@@ -103,7 +105,33 @@ def _setup_output_metrics(self, engine: Engine) -> Dict[str, Any]:
                 output_dict = {"output": output_dict}
 
             metrics.update({name: value for name, value in output_dict.items()})
-        return metrics
+
+        metrics_dict = {}  # type: Dict[Any, Union[str, float, numbers.Number]]
+
+        if key_tuple:
+
+            def key_tf(tag, name, *args):
+                return (tag, name) + args
+
+        else:
+
+            def key_tf(tag, name, *args):
+                return "/".join((tag, name) + args)
```

Comment:
Can't we use lambda functions ?

Reply:
I tried with that, flake8 gives this [warning](https://www.flake8rules.com/rules/E731.html).
Should I ignore that and add lambda only?

Reply:
Yes, you are right. Let's do that like that:
```python
def key_tuple(tag, name, *args):
    return (tag, name) + args

def key_str(tag, name, *args):
    return "/".join((tag, name) + args)

key_tf = key_tuple if key_tuple else key_str
```

Reply:
Yes thats better, fixed it.